flytekit/annotated/launch_plan.py

```diff
@@ -228,3 +229,36 @@ def get_registerable_entity(self) -> SdkLaunchPlan:
             version=settings.version,
         )
         return self._registerable_entity
+
+
+class ReferenceLaunchPlan(ReferenceEntity, LaunchPlan):
```

Comment:
can we get some docs on this? Especially as this is a user facing entity

Reply:
Sure.